easybuild/easyconfigs/m/meshtool/meshtool-20250401-GCCcore-13.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'meshtool'
+version = '20250401'
+
+homepage = 'https://bitbucket.org/aneic/meshtool'
+description = """Meshtool is a comand-line tool written in C++.
+ It is designed to apply various manipulations to volumetric meshes."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'openmp': True}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://bitbucket.org/aneic/meshtool',
+        'repo_name': 'meshtool',
+        'commit': '2b5c00311b3ac18d4e7b169a75aeadbc85fcc94f',
+    },
+}]
+checksums = ['ca35ddcc1be988283bdfb79ca314f3c75a2d001ed1bfab8c54e827d1bc4c4f4b']
+
```

```suggestion

builddependencies = [ 
    ('binutils', '2.40'),
]

```

Needed for the CI and also to build on systems that might not have binutils available.